pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[tool.poetry]
+name = "rero-ebooks"
+version = "0.2.0"
+description = "Invenio digital library framework."
+authors = ["RERO <software@rero.ch>"]
+license = "GNU Affero General Public License v3.0"
+
+[tool.poetry.dependencies]
+python = ">= 3.6, < 3.8"
+## Python packages dependencies (order matters)
+#------------------------------------------------------------------------------
+## Note: poetry evaluates the package constraints from the top to the bottom
+## and respects only the last constraint to install the package.
+## Thus the most strength constraint should be at the end.
+## Imagine the following package list for the foo package with the
+## most recent version equal to 5.0:
+## A foo Pipfile example:
+## ----------------------
+## foo = ">1.0" # first will install 5.0
+## foo = "==1.0" # then will downgrade to 1.0
+## foo = ">3.0" # finally will upgrade to 5.0
+## when the python code will be executed a conflict exception will be raised.
+#------------------------------------------------------------------------------
+# Default from Invenio
+Babel = ">=2.4.0"
+Flask-BabelEx = ">=0.9.3"
+
+## Third party invenio modules used by RERO ILS
+invenio-oaiharvester = {tag = "v1.0.0a4", git = "https://github.com/inveniosoftware/invenio-oaiharvester.git"}
+
+## Invenio 3.2.1 base modules used by RERO ILS
+# same as invenio metadata extras without invenio-search-ui
+invenio-indexer = ">=1.1.1,<1.2.0"
+invenio-jsonschemas = ">=1.0.1,<1.1.0"
+invenio-oaiserver = ">=1.1.1,<1.2.0"
+invenio-pidstore = ">=1.1.0,<1.2.0"
+invenio-records-rest = ">=1.6.4,<1.7.0"
+invenio-records-ui = ">=1.0.1,<1.1.0"
+invenio-records = ">=1.3.0,<1.4.0"
+
+## Default from Invenio
+invenio = {version = "==3.2.1", extras = ["base", "postgresql", "auth", "elasticsearch7" ]}
+uwsgi = ">=2.0"
+uwsgitop = ">=0.11"
+uwsgi-tools = ">=1.1.1"
+
+## RERO ILS specific python modules
+PyYAML = ">=5.3.1"
+isbnlib = ">=3.9.1"
+requests = ">=2.20.0"
+responses = "*"
+
+## Additionnal constraints on python modules
+# solves datetime serialize error
+# celery = "<4.3.0"
+
+## Deployment
+# sentry
+invenio-logging = { version = ">=1.2.0,<1.3.0", extras = ["sentry-sdk", "sentry"] }
+lxml = ">=3.5.0,<4.2.6"
+python-dotenv = "^0.13.0"
+
```

Comment:
why not uwsgi?

Reply:
This file is more or less a copy from `rero-ils` and `uwsgi` is defined in line 43.